MetabolicAtlas/MetabolicAtlas
# Feature/1334 add new dataset for map overlay

**Related issue(s) and PR(s)**  
This PR closes #1334

Adds documentation for the a new dataset for Yeast-GEM.

**Type of change**  
 
**List of changes made**  
- Update documentation to include description of new dataset
- Update `About -> Platform -> Citation -> Research-driven features` to include paper related to new dataset
- Clarify separation between `DATA_TYPES_COMPONENTS` and local replication in component `DataOverlay`.

**Screenshot of the fix**  
![01-citation](https://user-images.githubusercontent.com/124560665/230058954-00db1fc1-13aa-42f1-8e8c-0af8720dae9f.png)
![02-documentation'](https://user-images.githubusercontent.com/124560665/230058942-9bd22d77-49dd-431b-8a85-c03bd4df9062.png)

**Testing**  
- Can be tested without but is related to: https://github.com/MetabolicAtlas/data-files/pull/34
- Test with new data `build-stack && start-stack`
- http://localhost/documentation#data-overlay
- http://localhost/about/platform#citation
- Make sure the text placement makes sense
- Make sure the references look correct

**Further comments**  


**Definition of Done checklist**
